qiskit/providers/ibmq/job/schema.py

```diff
@@ -36,7 +36,6 @@
     'error': '_error',
     'name': '_name',
     'timePerStep': '_time_per_step',
-    'shots': '_api_shots',
```

Comment:
Can you verify that when retrieving older jobs, they won't have the `shots` attribute?

Reply:
According to the API, `shots` would still be present in the root level attributes when retrieving older jobs. But, it would not be present in the root level for newer jobs that only submitted `shots` within the obj. As a result, I re-added the `'shots': '_api_shots'` field back into `FIELD_MAPS`.